test/e2e/validators.go

```diff
@@ -50,6 +49,24 @@ func cleanupOptions(ctx *framework.TestCtx) *framework.CleanupOptions {
 	}
 }
 
+// clientGet is a wrapper around f.Client.Get that performs retries in case of retryable errors
+func clientGet(key types.NamespacedName, obj runtime.Object) error {
+	f := framework.Global
+
+	return wait.PollImmediate(1*time.Second, 60*time.Second, func() (bool, error) {
+		err := f.Client.Get(goctx.TODO(), key, obj)
+
+		if err == nil {
+			return true, nil
```

Comment:
based on the way we have been managing errors , can u keep this one as:
```
if err := f.Client.Get(goctx.TODO(), key, obj); err != nil {
 return false, err
}
```

and at the end of the func do a `return false, nil` ?

Reply:
@qu1queee I changed it.

I like this golang pattern (if with expression and condition) but need to get used to using it. :-)

Based on my understanding, one has to do `return true, nil` at the end. Otherwise one would not stop the loop. See https://godoc.org/k8s.io/apimachinery/pkg/util/wait#PollImmediate